geomesa-arrow/geomesa-arrow-jts/src/main/java/org/locationtech/geomesa/arrow/vector/WKBGeometryVector.java

```diff
@@ -0,0 +1,109 @@
+/***********************************************************************
+ * Copyright (c) 2013-2020 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.arrow.vector;
+
+import org.apache.arrow.memory.BufferAllocator;
+import org.apache.arrow.vector.Float8Vector;
+import org.apache.arrow.vector.VarBinaryVector;
+import org.apache.arrow.vector.complex.AbstractContainerVector;
+import org.apache.arrow.vector.complex.FixedSizeListVector;
+import org.apache.arrow.vector.types.pojo.ArrowType;
+import org.apache.arrow.vector.types.pojo.Field;
+import org.apache.arrow.vector.types.pojo.FieldType;
+import org.locationtech.geomesa.arrow.vector.impl.AbstractPointVector;
+import org.locationtech.jts.geom.Geometry;
+import org.locationtech.jts.io.ParseException;
+import org.locationtech.jts.io.WKBReader;
+import org.locationtech.jts.io.WKBWriter;
+
+import javax.annotation.Nullable;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Catch-all for storing instances of Geometry as WKB
+ */
+public class WKBGeometryVector implements GeometryVector<Geometry, VarBinaryVector> {
+  private VarBinaryVector vector;
+  private WKBWriter writer = new WKBWriter();
+  private WKBReader reader = new WKBReader();
+
+  public static FieldType createFieldType(Map<String, String> metadata) {
+    return new FieldType(true, ArrowType.Binary.INSTANCE, null, metadata);
+  }
+
+  public WKBGeometryVector(String name, BufferAllocator allocator, @Nullable Map<String, String> metadata) {
+    // JNH: How do I use the metadata?
```

Comment:
i think as long as you pass it through like this then you don't need to do anything else with it

Reply:
Got it!